crates/iota-sdk/src/apis/coin_read.rs

```diff
@@ -313,4 +313,22 @@ impl CoinReadApi {
     pub async fn get_total_supply(&self, coin_type: impl Into<String>) -> IotaRpcResult<Supply> {
         Ok(self.api.http.get_total_supply(coin_type.into()).await?)
     }
+
+    /// Get the IOTA circulating supply summary.
+    ///
+    /// # Examples
+    ///
+    /// ```rust,no_run
+    /// use iota_sdk::IotaClientBuilder;
+    ///
+    /// #[tokio::main]
+    /// async fn main() -> Result<(), anyhow::Error> {
+    ///     let iota = IotaClientBuilder::default().build_localnet().await?;
```

Comment:
Do we want localnet here?

Reply:
I don't think it matters, it's not ran, just examples. Would you prefer testnet?

Reply:
Not super important, but I think that's generally better since most people would probably start there instead of a localnet

Reply:
Ok I will change it in more places then

Reply:
done